packages/polymath-issuer/src/pages/providers/data.js

```diff
@@ -67,6 +67,14 @@ export const categories: Array<SPCategory> = [
       'security tokens to be held for safekeeping and minimize the risk of theft or loss. Alternatively, ' +
       'you and/or your Investors can elect to self custody funds and security tokens.',
   },
+  {
+    id: 5,
+    title: 'Token Sale Platform',
+    desc:
+      "Apply for Custody services for the funds you raised and/or your investors' " +
```

Comment:
This description should match what is written down in issue 677

Reply:
Thanks. Will fix shortly